### ISSUE TITLE
glTexStorage2D( GL_TEXTURE_CUBE_MAP )

### ISSUE THREAD

**Issue by Dark-Photon** [danginsburg/opengles3-book#17]
According to the GLES 3.0 spec, it appears that the following GL call is valid for cube maps (see section 3.8.4 on pg. 137):

glTexStorage2D( GL_TEXTURE_CUBE_MAP, ... )

This usage is also described in GL-land (ARB_texture_storage ext spec, OpenGL Programming Guide, etc.). 

However, it appears that the OpenGL ES 3.0 Programming Guide is not consistent with the GLES 3.0 spec here. It does not list GL_TEXTURE_CUBE_MAP as a valid option for target, and it lists the face enums (GL_TEXTURE_CUBE_MAP_{POSITIVE,NEGATIVE}_{X,Y,Z}) as valid options for target.

**Comment by danginsburg** [comment]
You are correct, this is an error in the book occurring on page 276 in the description of <target> for glTexStorage2D.  Thanks for reporting it.
